<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Ajay3147361" name="lalaTestNGTest" tests="23" failures="2" timestamp="30 Nov 2018 10:22:52 GMT" time="4.333" errors="0">
  <testcase name="verifyFutureOrder" time="1.477" classname="com.lala.test.PlaceOrderTest">
    <failure type="java.lang.AssertionError" message="Test is failed, Error while placing order, Please check by placing order manually.">
      <![CDATA[java.lang.AssertionError: Test is failed, Error while placing order, Please check by placing order manually.
	at com.lala.test.PlaceOrderTest.verifyFutureOrder(PlaceOrderTest.java:73)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- verifyFutureOrder -->
  <testcase name="verifyNewOrder" time="0.897" classname="com.lala.test.PlaceOrderTest"/>
  <testcase name="verifyNewOrderWithInvalidPayload" time="0.027" classname="com.lala.test.PlaceOrderTest"/>
  <testcase name="verifyFetchOrderForNewOrder" time="0.304" classname="com.lala.test.FetchOrderTest"/>
  <testcase name="verifyFetchOrderWhenOrderDoesNotExist" time="0.023" classname="com.lala.test.FetchOrderTest"/>
  <testcase name="verifyFetchOrderWhenOrderExist" time="0.024" classname="com.lala.test.FetchOrderTest"/>
  <testcase name="verifyDriveToTakeAlreadyCancelledOrder" time="0.043" classname="com.lala.test.DriveToTakeOrderTest"/>
  <testcase name="verifyDriveToTakeAlreadyCompletedOrder" time="0.033" classname="com.lala.test.DriveToTakeOrderTest"/>
  <testcase name="verifyDriveToTakeAlreadyOnGoingOrder" time="0.032" classname="com.lala.test.DriveToTakeOrderTest"/>
  <testcase name="verifyDriveToTakeExistingOrder" time="0.022" classname="com.lala.test.DriveToTakeOrderTest">
    <failure type="java.lang.AssertionError" message="expected [200] but found [422]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [422]
	at com.lala.test.DriveToTakeOrderTest.verifyDriveToTakeExistingOrder(DriveToTakeOrderTest.java:75)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- verifyDriveToTakeExistingOrder -->
  <testcase name="verifyDriveToTakeOnOrderDoesNotExist" time="0.019" classname="com.lala.test.DriveToTakeOrderTest"/>
  <testcase name="verifyDriveToTakeOrder" time="0.445" classname="com.lala.test.DriveToTakeOrderTest"/>
  <testcase name="verifyDriveToCompleteForCancelOrder" time="0.043" classname="com.lala.test.DriveToCompleteOrderTest"/>
  <testcase name="verifyDriveToCompleteForCompletedOrder" time="0.046" classname="com.lala.test.DriveToCompleteOrderTest"/>
  <testcase name="verifyDriveToCompleteForNewOrder" time="0.431" classname="com.lala.test.DriveToCompleteOrderTest"/>
  <testcase name="verifyDriveToCompleteForNonExistingOrder" time="0.03" classname="com.lala.test.DriveToCompleteOrderTest"/>
  <testcase name="verifyDriveToCompleteForOnderOnAssignedState" time="0.051" classname="com.lala.test.DriveToCompleteOrderTest"/>
  <testcase name="verifyCancelExistingOrder" time="0.049" classname="com.lala.test.CancelOrderTest"/>
  <testcase name="verifyCancelNewOrder" time="0.168" classname="com.lala.test.CancelOrderTest"/>
  <testcase name="verifyCancelNonExistingOrder" time="0.026" classname="com.lala.test.CancelOrderTest"/>
  <testcase name="verifyCancelOrderForAlreadyCancelledOrder" time="0.029" classname="com.lala.test.CancelOrderTest"/>
  <testcase name="verifyCancelOrderForCompletedOrder" time="0.029" classname="com.lala.test.CancelOrderTest"/>
  <testcase name="verifyCancelOrderForOnGoingOrder" time="0.029" classname="com.lala.test.CancelOrderTest"/>
</testsuite> <!-- lalaTestNGTest -->
